files/nrpe/check_openstack.py

```diff
@@ -1301,6 +1358,15 @@ def main():
   except CinderServiceDisabledException as e:
     print(e)
     exit_with_stats(NAGIOS_STATE_WARNING)
+  except NeutronL3AgentsUnknown as e:
```

Comment:
This might not be the moment, but also e is a variable name too short, and the full extension of the name "error" is not that long either. But since right before and after is used e, maybe we can keep the same for consistency. But this also make our work longer in the future to fix this styling. So just a comment, no need to act.

Reply:
For consistency reasons I didn't make any changes for this